README.adoc

```diff
@@ -43,8 +41,32 @@ If you already have Rust installed, make sure you're using the latest version by
 rustup update
 ----
 
+=== Install PoC-4 on "Alexander" Testnet
+
+Build Polkadot PoC-4 by cloning this repository and running the following commands from the root directory of the repo:
+
+```bash
+git checkout v0.4
+./scripts/init.sh
+./scripts/build.sh
```

Comment:
```suggestion
./scripts/build.sh
cargo build --release
```
it is still necessary to call cargo build, `build.sh` builds just the runtime Wasm